robotiq_cpp_control/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(robotiq_cpp_control)
+
```

Comment:
Can you say why the flags below are required.  Typically, these aren't included in the `CMakeLists.txt`